Makefile

```diff
@@ -48,14 +47,26 @@ test:
 
 .PHONY: all test clean
 
+## Update Tag and push to github 
 tag:
-	@echo "Bump version :- $(last_tag) -> $(new_version)"
-	@sed -i "s/version.*=.*/version = '$(new_version)'/" datamegh/__init__.py && \
-		git add datamegh/__init__.py && \
-		git commit -m "Update version to $(new_version)" -m "[skip ci]" && \
-		git remote add origin-pusher https://${GITHUB_OAUTH_TOKEN}@github.com/leapfrogtechnology/datamegh.git || true && \
-		git push origin-pusher ${TRAVIS_BRANCH} --tags
+	echo "Bump version :- $(last_tag) -> $(new_version)"
+	git remote set-url origin https://${GITHUB_TOKEN}@github.com/leapfrogtechnology/datamegh.git
+	git config --global user.name "leapfrog Bot"
+	git config --global user.email "devops@lftechnology.com"
```

Comment:
Really this is the email address of the bot? Don't we have a email address like `bot@lftechnology.com`?